src/skin/popup.css

```diff
@@ -44,7 +44,12 @@ body {
     src: url("./fonts/Chunk.ttf");
 }
 
-a { text-decoration: none }
+a {
```

Comment:
Since we also include popup.css on the options page (to style the sliders under the Tracking Domains tab), this anchor style definition affects all options page links.

We should either make this CSS rule selector more specific so that it only applies to links in the popup, or we should take the parts of popup.css that are needed on the options page and move them, perhaps to `src/skin/toggle-switch.css`.

Reply:
I made a small change in the src/skin/popup.css file and it seems to work properly now.
![Screen Shot 2020-05-06 at 5 23 44 PM](https://user-images.githubusercontent.com/38918932/81231441-26c31280-8fc1-11ea-9ba2-cc61db5e601c.png)

![Screen Shot 2020-05-06 at 5 22 48 PM](https://user-images.githubusercontent.com/38918932/81231543-583bde00-8fc1-11ea-98b5-1d1eac9c14e2.png)

![Screen Shot 2020-05-06 at 5 23 21 PM](https://user-images.githubusercontent.com/38918932/81231550-5bcf6500-8fc1-11ea-85f1-223b13672f57.png)

![Screen Shot 2020-05-06 at 5 21 20 PM](https://user-images.githubusercontent.com/38918932/81231602-6be74480-8fc1-11ea-9439-9822092e96e3.png)

![Screen Shot 2020-05-06 at 5 22 09 PM](https://user-images.githubusercontent.com/38918932/81231619-70abf880-8fc1-11ea-9acd-9a5cac62488b.png)